ui/pages/confirmations/components/confirm/title/title.tsx

```diff
@@ -10,6 +10,44 @@ import {
 import { useI18nContext } from '../../../../../hooks/useI18nContext';
 import { currentConfirmationSelector } from '../../../../../selectors';
 import { Confirmation } from '../../../types/confirm';
+import useAlerts from '../../../../../hooks/useAlerts';
+import { getHighestSeverity } from '../../../../../components/app/alerts-system/utils';
+import SecurityAlertBanner from '../../../../../components/app/alerts-system/security-alert-banner/security-alert-banner';
+
+function ConfirmBannerAlert({ ownerId }: { ownerId: string }) {
+  const t = useI18nContext();
+  const { generalAlerts } = useAlerts(ownerId);
+  const hasMultipleAlerts = generalAlerts.length > 1;
+
+  if (generalAlerts.length === 0) {
+    return null;
+  }
+
+  const singleAlert = generalAlerts[0];
+  const highestSeverity = hasMultipleAlerts
+    ? getHighestSeverity(generalAlerts)
+    : singleAlert.severity;
+  return (
+    <Box marginTop={4}>
+      <SecurityAlertBanner
```

Comment:
Since this `SecurityAlertBanner` is ultimately the counterpart to inline alert for general alerts, then I'm not sure if `Security` is accurate as it will display any general alert in any of the confirmations regardless of whether it is security focused.

Would it add clarity to rename the `SecurityAlertBanner` to something else such as `GeneralAlert` for example?